### PR TITLE
[2.1] Add "expand" option for stretch aspect, no more black bars

### PR DESCRIPTION
**Pull request by jkb0o**
This pr provides "expand" option for Display/stretch_aspect so with "2d" stretch mode there is no more black bars:

[hd video](https://www.youtube.com/watch?v=wFF-miUvkYQ)
example

### CODE REVIEW

File: scene/main/scene_main_loop.h [hunk 1]
```diff
@@ -66,6 +66,7 @@ class SceneTree : public MainLoop {
 		STRETCH_ASPECT_KEEP,
 		STRETCH_ASPECT_KEEP_WIDTH,
 		STRETCH_ASPECT_KEEP_HEIGHT,
+        STRETCH_ASPECT_EXPAND,
```
Comment (bojidar-bg) [inline scene/main/scene_main_loop.h]: Use tabs.

**Comment by jjay** [comment]
fix indentation

**Comment by RandomShaper** [comment]
Please test `VisibilityNotifier2D`. Setup a project so that the black bars would be very big if using `keep` and move a very small sprite around checking if enter/exit viewport notifications work as expected.

Furthermore, if that works correctly with `expand`, it's `keep` what would need correction so it doesn't notify enter below the black bars. But that wouldn't be your responsibility. :)

Do you know what I mean?

**Comment by jjay** [comment]
@RandomShaper I know what you mean. I just did some tests and `VisibilityNotifier2D` has issues with `keep`, and works fine with `expand`.

**Comment by HummusSamurai** [comment]
Can't believe there was no "keep" option before!!!

**Comment by RandomShaper** [comment]
@jjay, thanks. I'll have a look, but this PR is fine then.

**Review by RandomShaper — APPROVED** [review]

**Pull request merged**